cypress/objects/timeLoginPage.js

```diff
@@ -0,0 +1,38 @@
+class TimeLoginPage {
+    visit() {
+        cy.visit('/')
+    }
+
+    getURL() {
+        return cy.url()
+    }
+
+    getPageTitle() {
+        return cy.get(".page__title").contains('Timesheets')
+    }
+
+    getCalendarVisable() {
+        return cy.get('.calendar')
+    }
+
+    getCalendarToday() {
+        return cy.get('.calendar--today')
+    }
+
+    getTimeEntryForm() {
+        return cy.get('.tile.form')
+    }
+
+    getUserInfoTitle() {
+        return cy.get('.user-info__title').contains('Ugne Lekstutiene')
```

Comment:
I'd drop the 'contains' part and add it in the test cases instead of this function's definition :)